channels-rke2.yaml

```diff
@@ -617,7 +617,7 @@ releases:
       <<: *featureVersions-v1-22-11-rke2r1
   - version: v1.22.15+rke2r1
```

Comment:
```
channels:
  - name: default
    latest: v1.22.15+rke2r1
```
Would we need to update the default version for 2.7 now that v1.22.x won't show up?

Reply:
The section you highlighted is deprecated but kept for backward compaticitbiy.
Now the default version is read from the `appDefaults` section, which indicates that 1.24.x is the default for rancher 2.7.0 
```
      - appVersion: '>= 2.6.7-0 < 2.7.100-0'
        defaultVersion: '1.24.x'
```

Reply:
ok, this makes sense